### PR TITLE
feat: send email on schedule edit by admin

### PR DESCRIPTION
**Pull request by akshay2000saxena**
## Implementation description. How did you make this change?
* 

Screen Shot 2022-02-12 at 11 40 42 AM

## Steps to test
1.

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages follow conventional commits and are descriptive. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s) 
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [x] The appropriate tests if necessary have been written

### CODE REVIEW

File: backend/typescript/services/interfaces/schedulingService.ts [hunk 1]
```diff
@@ -34,20 +34,6 @@ interface ISchedulingService {
    */
   getSchedulings(): Promise<Array<SchedulingDTO>>;
```
Comment (hanlinc27) [inline backend/typescript/services/interfaces/schedulingService.ts:36]: Can we add in the correct `sendScheduledDonationEmail` into the interface here

File: backend/typescript/services/implementations/schedulingService.ts [hunk 2]
```diff
@@ -529,21 +387,175 @@ class SchedulingService implements ISchedulingService {
...
+     </html>
+      `;
+
+      this.emailService.sendEmail(
+        currDonor.email,
+        updated
+          ? `DONATION EDITED - Donation on ${startDayString}`
+          : "Your Donation Information",
+        emailBody,
+      );
```
Comment (hanlinc27) [inline backend/typescript/services/implementations/schedulingService.ts:550]: doesn't this no longer include the changes for sending the correct email copy to **admin** for new donations?
https://github.com/uwblueprint/community-fridge-kw/pull/123

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

File: backend/typescript/services/interfaces/schedulingService.ts (hunk 1 above)
Comment (akshay2000saxena) [reply backend/typescript/services/interfaces/schedulingService.ts]: Done

File: backend/typescript/services/implementations/schedulingService.ts (hunk 2 above)
Comment (akshay2000saxena) [reply backend/typescript/services/implementations/schedulingService.ts]: Looks like my branch was outdated and didn't have that PR, updated now

**Review by hanlinc27 — APPROVED** [review]
:shipit:

**Pull request merged**